src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -16,8 +16,10 @@
 
 		<!-- App Identifier -->
 		<ApplicationId>com.companyname.mauiapp._1</ApplicationId>
+		<ApplicationId Condition="$(TargetFramework.Contains('-windows'))">8B51DC95-6D07-4C39-BC6C-3BFE96E8A7EA</ApplicationId>
```

Comment:
`Guid.NewGuid()` needed, and this one is uppercase for some reason?

Reply:
@mattleibow - though, THIS guid here I don't see listed in `template.json`. Does it need to be added there so that it gets a unique GUID upon creation?

Reply:
https://github.com/dotnet/maui/blob/6ce55cf7d8f148ac7edb1e4cecf8ac93a7bc54fb/src/Templates/src/templates/maui-blazor/.template.config/template.json#L52

Reply:
Oh weird - it seems that the MAUI Regular and MAUI Blazor templates use one GUID the same, and then one GUID different. That's why I missed it!

MAUI:
https://github.com/dotnet/maui/blob/6ce55cf7d8f148ac7edb1e4cecf8ac93a7bc54fb/src/Templates/src/templates/maui-mobile/.template.config/template.json#L55-L56

Blazor:
https://github.com/dotnet/maui/blob/6ce55cf7d8f148ac7edb1e4cecf8ac93a7bc54fb/src/Templates/src/templates/maui-blazor/.template.config/template.json#L51-L52

So this is fine, though a bit odd, and not part of this PR.